gradle/versions.gradle

```diff
@@ -15,11 +15,16 @@
  */
 
 apply plugin: 'com.github.ben-manes.versions'
-apply plugin: 'se.ascp.gradle.gradle-versions-filter'
 
-versionsFilter {
-    exclusiveQualifiers = ['alpha', 'beta', 'rc', 'cr', 'm', 'ea']
-    log = false
+dependencyUpdates.configure {
+
+    def isNonStable = { String version ->
+        return ['alpha', 'beta', 'rc', 'cr', 'm', 'ea'].any { keyword -> version.toUpperCase().contains(keyword.toUpperCase())}
```

Comment:
do we need more complex check with regex (as it was in version plugin)?
https://github.com/janderssonse/gradle-versions-filter-plugin/blob/c2682188f0f4fe5b36d72f5cb9561a9222ca4e50/src/main/kotlin/se/ascp/gradle/GradleVersionsFilterPlugin.kt#L135

Reply:
I don't think so